src/sentry/static/sentry/app/components/charts/components/tooltip.jsx

```diff
@@ -21,22 +21,31 @@ function valueFormatter(value) {
 
 function getFormatter({filter, isGroupedByDate, truncate, formatAxisLabel, utc}) {
   const getFilter = seriesParam => {
-    const value = seriesParam.data[1];
+    // Releases can be a series with an empty `data`, but real series will
+    // have `data` as a tuple of (key, value)
+    const value = seriesParam.data && seriesParam.data.length && seriesParam.data[1];
```

Comment:
So releases will be `[]` and otherwise we have `[val1, val2]`. What's the case where data won't be truthy?

Reply:
It's possible to define a series without `data` at all, (like in the case of releases), and charts would render fine

Reply:
The comment above implies in the case of releases it's an empty list i.e. `[]` rather than falsy, perhaps its worth updating the comment if it's actually expected to be `null` or `undefined` as well?